app/src/test/java/org/simple/clinic/summary/PatientSummaryViewRendererTest.kt

```diff
@@ -36,7 +36,7 @@ class PatientSummaryViewRendererTest {
   private val defaultModel = PatientSummaryModel.from(ViewExistingPatient, UUID.fromString("6fdf088e-f6aa-40e9-9cc2-22e197b83470"))
   private val ui = mock<PatientSummaryScreenUi>()
 
-  private val uiRenderer = PatientSummaryViewRenderer(ui)
+  private val uiRenderer = PatientSummaryViewRenderer(ui, modelUpdateCallback = null)
```

Comment:
Can we make a mock object instead of `null` and verify that that the listener was invoked whenever the model changes with the correct model?